src/templates/contact-form.twig

```diff
@@ -0,0 +1,132 @@
+                                <label class="form-check-label" for="gridRadios3">
+                                    Three seats
+                                </label>
+                            </div>
+                            <div class="form-check">
+                                <input class="form-check-input" id="gridRadios4" name="Seats" type="radio" value="four">
+                                <label class="form-check-label" for="gridRadios4">
+                                    Four seats
+                                </label>
+                            </div>
+                            <div class="form-check">
+                                <input class="form-check-input" id="gridRadios5" name="Seats" type="radio" value="five">
+                                <label class="form-check-label" for="gridRadios5">
+                                    Five seats
+                                </label>
+                            </div>
+                        </div>
+                    </fieldset>
+                </div>
+
+                <div class="form-row">
+                    <div class="form-group col-md-6">
+                        <label for="inputNote">Note</label>
+                        <textarea class="form-control" id="inputNote" name="Note" rows="3"></textarea>
+                    </div>
+                </div>
+
+                <div class="form-check mb-5">
+                    <input class="form-check-input" id="gridCheck" name="Agreement" required type="checkbox">
+                    <label class="form-check-label" for="gridCheck">
+                        I agree to the processing of personal data
+                    </label>
+                </div>
+
+                <button class="btn btn-primary btn-lg" type="submit">Send</button>
+                <input name="_next" type="hidden" value="https://fictive-hipster-cafe.surge.sh/reservation-send"/>
+            </form>
+        </div>
+    </div>
+
+    <script>
+        var phoneSearch = document.getElementById("inputPhone"); // definice funkce, ktera v documentu najde element podle Id
+        var mailSearch = document.getElementById("inputE-mail");
+
+        function setRequired() {
+            var phoneCheck = phoneSearch.value;
+            var mailCheck = mailSearch.value;
+
+            if (phoneCheck || mailCheck) {
+                phoneSearch.removeAttribute("required"); //odebira atribut required, pokud je jedno z poli vyplnenych
+                mailSearch.removeAttribute("required");
+            } else {
+                phoneSearch.setAttribute("required", "true"); //hodnota atributu required muze byt tez ""
+                mailSearch.setAttribute("required", "true");
+            }
+        }
+
+        phoneSearch.addEventListener("input", setRequired);
+        mailSearch.addEventListener("input", setRequired);
```

Comment:
I když to tady výkon nezabije, bude lepší dát jen jeden listener a využít [event delegation](https://gomakethings.com/why-event-delegation-is-a-better-way-to-listen-for-events-in-vanilla-js/#web-performance).